Kitodo/src/main/java/de/sub/goobi/helper/BatchStepHelper.java

```diff
@@ -220,10 +217,14 @@ private void saveStep() {
      * Error management for single.
      */
     public String reportProblemForSingle() {
-
         this.myDav.uploadFromHome(this.currentStep.getProcess());
-        reportProblem();
-        this.problemMessage = "";
+        serviceManager.getWorkflowService().setProblem(getProblem());
+        try {
+            this.currentStep = serviceManager.getWorkflowService().reportProblem(this.currentStep, this.problemTask);
+        } catch (DataException e) {
+            logger.error("Problem couldn't be reported: " + e);
```

Comment:
2 things:
1) If a `DataException` is thrown is it a good idea to go on?
2) Did this kind of error logging work with casting exception object to string message?

Reply:
1. Partially yes, partially no. Method saveStep shouldn't be called but it shoul give back the list of tasks. 
2. Yes, the whole exception will be printed out.